openedx/features/badging/tests/test_model_badge.py

```diff
@@ -39,29 +29,10 @@ def test_save_badge_normal(self):
                       date_created=None)
         self.assertEqual(badge.save(), None)
 
-    def test_badges_empty(self):
+    def test_get_badges_json(self):
         """
-        Test get badges if no badge is created
+        Test get badges json method
         """
-        expected_result = {}
-        returned_result = Badge.get_badges(community_type="no_match_")
+        expected_result = '[]'
+        returned_result = Badge.get_badges_json(badge_type="no_match_")
         self.assertEqual(expected_result, returned_result)
-
-    def test_all_badges(self):
-        """
```

Comment:
Let's please add a test for badge ordering.

Reply:
done